packages/create-nx-plugin/bin/create-nx-plugin.ts

```diff
@@ -43,34 +43,44 @@ export const yargsDecorator = {
 
 const nxVersion = require('../package.json').version;
 
-function determinePluginName(parsedArgs: CreateNxPluginArguments) {
+async function determinePluginName(
+  parsedArgs: CreateNxPluginArguments
+): Promise<string> {
   if (parsedArgs.pluginName) {
-    return Promise.resolve(parsedArgs.pluginName);
+    return parsedArgs.pluginName;
   }
 
-  return enquirer
-    .prompt([
-      {
-        name: 'pluginName',
-        message: `Plugin name                        `,
-        type: 'input',
-        validate: (s) => (s.length ? true : 'Name cannot be empty'),
-      },
-    ])
-    .then((a: { pluginName: string }) => {
-      if (!a.pluginName) {
-        output.error({
-          title: 'Invalid name',
-          bodyLines: [`Name cannot be empty`],
-        });
-        process.exit(1);
-      }
-      return a.pluginName;
-    });
+  const results = await enquirer.prompt<{ pluginName: string }>([
+    {
+      name: 'pluginName',
+      message: `Plugin name                        `,
+      type: 'input',
+      validate: (s_1) => (s_1.length ? true : 'Plugin name cannot be empty'),
+    },
+  ]);
+  return results.pluginName;
+}
+
+async function determineCreatePackageName(
+  parsedArgs: CreateNxPluginArguments
+): Promise<string> {
+  if (parsedArgs.createPackageName) {
+    return parsedArgs.createPackageName;
+  }
+
+  const results = await enquirer.prompt<{ createPackageName: string }>([
+    {
+      name: 'createPackageName',
+      message: `Create package name (optional)           `,
```

Comment:
Followup: Adjust this message to say leave it blank if you don't want one

```suggestion
      message: `Create a package which can be used by npx to create a new workspace (Leave blank to not create this package)`,
```